torchvision/prototype/models/efficientnet.py

```diff
@@ -94,7 +94,17 @@ class EfficientNet_B1_Weights(WeightsEnum):
             "acc@5": 94.186,
         },
     )
-    default = ImageNet1K_V1
+    ImageNet1K_V2 = Weights(
+        url="https://download.pytorch.org/models/efficientnet_b1-c2e40cbf.pth",
+        transforms=partial(ImageNetEval, crop_size=224, resize_size=232, interpolation=InterpolationMode.BILINEAR),
```

Comment:
Blocking: As discussed offline the network was trained with an inference resolution 224x224 in mind instead of the 240x240 required by B1. This means that you could push this model to even higher accuracies! So first of all change your inference configuration to `--val-crop-size 240 --val-resize-size 240`.

Check a few more `--train-crop-size` values for the FixRes mitigations and keep the one that gives you the best trained model at 240 inference. Once you have the final configuration, do 2 more runs and then select the best possible model and tune `--val-resize-size`.